generator/recipe.go

```diff
@@ -21,10 +20,6 @@ type Template struct {
 	Processors map[string]string
 }
 
-var templateFuncs = map[string]interface{}{
```

Comment:
I wonder as templateFuncs is not used anywhere in codebase, linting was showing it as unused var

Reply:
Hmm, strange, I used it template string. Looking into it, maybe changes were made after my pr. On it

Reply:
@scortier left a comment on the commit you pushed. In the generator package. I will make another PR to fix that if there was no reason to remove that??

Reply:
Checked it, related to some conflict issue, updated the code 👍

Reply:
It throws error right?? while running meteor gen command?? Let me fix it first, since it is in main branch and this pr might take time before getting merged

Reply:
Sure go ahead, but i think we can merge this pr soon as not much breaking changes are made , wdyt ?